lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -33,7 +33,7 @@
 </script>
 
 <div class="batch-enrollment membership-section">
-  <h2> ${_("Batch Enrollment")} </h2>
+  <h3 class="hd hd-2">${_("Batch Enrollment")}</h3>
```

Comment:
Looking at your sandbox, I think you need some additional differentiation between h3 and what follows it. That is to say, I expect more kerning or space, etc after "Batch Enrollment". It looks compressed to me.